docs/ipv6.md

```diff
@@ -37,13 +37,14 @@ If this doesn't work, figure it out.  It may be necessary to add an firewall rul
 
 ### Step 2 — Update Docker's Init To Enable IPv6 Support
 
-Copy the system's existing docker file and append the `--ipv6` argument to the end of the command line:
 
-    sed -e 's:^\(ExecStart.*\):\1 --ipv6:' /lib/systemd/system/docker.service | tee /etc/systemd/system/docker.service
+Append the `--ipv6` argument to the `DOCKER_OPTS` variable in:
+
+    /etc/default/docker
 
 Reload the daemon and restart docker so that it takes affect:
 
-    systemctl daemon-reload && systemctl restart docker.service
+    service docker restart
```

Comment:
If using systemd, people need to use systemctl.  Ubuntu provides `service` binary to manage upstart services and maybe will soon wrap systemd interface.

Probably need service and systemctl commands for upstart and systemd respectively.